cylc/uiserver/main.py

```diff
@@ -110,6 +113,100 @@ def validate(self, obj, value):
 
 class CylcUIServer(Application):
 
+    # TODO: Add a link to the access group table mappings in cylc documentation
+    AUTH_DESCRIPTION = '''
+            Authorization can be granted at operation (mutation) level, i.e.
+            specifically grant user access to execute Cylc commands, e.g.
+            `play`, `pause`, `edit`, `trigger` etc. For your convenience,
+            these operations have been mapped to access groups `READ`,
+            `CONTROL` and `ALL`.
+
+            To remove permissions, prepend the access group or operation with
+            `!`.
+
+            Permissions are additive but negated permissions take precedence
+            above additions.
+
+            .. note::
+
+            Any authorization permissions granted to a user will be
+            applied to all workflows.
+    '''
+
+    site_authorization = Dict(
+        config=True,
+        help='''
+            Dictionary containing site limits and defaults for authorization.
+            This configuration should be placed only in the site set
+            configuration file and not the user configuration file (use
+            ``c.UIServer.user_authorization`` for user defined authorization).
+
+            If this configuration is empty, site authorization defaults to no
+            configuarble authorization and users will be unable to set any
+            authorization.
+        ''' + AUTH_DESCRIPTION + '''
+        Example Configuration:
+
+        .. code-block:: python
+
+        c.UIServer.site_authorisation = {
```

Comment:
Nit-pick, but there's a mix of authorise, authorization, authorisation in our code & docs. Not sure if we should use just one of use them interchangeably.

Reply:
Yes oops, thought I'd changed them all to the American spelling for consistency. Wonder if it would be worth supporting both spelling types for the config though?

Reply:
I'm more used to seeing the American spelling for color (HTML standard) and authorization (W3C standards, Microsoft Active Directory, AWS, OAuth, SAML, Kubernetes, Docker), but I'm used to seeing both in documentation, websites, etc :man_shrugging: In Cylc 7 I think we only had authorisation, and there might be a Python module authorise or authorisation in cylc-flow. Maybe check with others in Element to see if we have to change, otherwise we can just keep both I guess.